cupyx/scipy/sparse/_coo.py

```diff
@@ -472,7 +472,7 @@ def toarray(self, order=None, out=None):
         return self.tocsr().toarray(order=order, out=out)
 
     def tocoo(self, copy=False):
-        """Converts the matrix to COOdinate format.
+        """Converts the matrix to coordinate format.
```

Comment:
```suggestion
        """Converts the matrix to COOrdinate format.
```

Copied from https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.coo_matrix.tocoo.html#scipy.sparse.coo_matrix.tocoo